### DISCUSSION TITLE
Export results and get it back into endnote

### DISCUSSION THREAD

**Discussion by taviaevans** [asreview/asreview#638] [Q&A]
Hi, 

I was wondering if (how) I can get the exported results back into endnote?

Thanks!

**Comment by boer0107-zz** [comment]
We are working on a .ris export, so that you can import your results into a reference manager like Endnote. This feature isn't available in the current release of ASR (0.17).
We do have a workaround, which does require some R skills: https://github.com/asreview/asreview/issues/225. Also notice the remark at the end of the workaround: it does not seem to work for Endnote! This means you would probably need a second workaround, importing the bib-files into Zotero and exporting them again. This should finaly result in a file that can be imported into Endnote.

**Reply by Rensvandeschoot** [reply]
@boer0107 does Endnote accept `.tsv` files?

**Reply by boer0107-zz** [reply]
It does, however I would not recommend it unless necessary. I have never done so myself.
You can watch this video for instructions: https://www.youtube.com/watch?v=FFmN2xQ-Bqo

**Comment by ottomattas** [comment]
Hello @taviaevans. Indeed, as @boer0107 said, we are working on a feature that would allow you to export in .ris format from ASReview. As a result, we should be able to streamline the workflow with Endnote.
As soon as this is implemented, I will also come back here to let you know so you can try it out.

**Comment by ckolar** [comment]
Zotero user hoping for the same.  Thanks for such a great tool.

**Reply by ottomattas** [reply]
Thanks for bringing your wish out @ckolar. The feature is also being implemented with Zotero in mind. Hopefully, we can give you good news rather sooner than later.

**Reply by retorquere** [reply]
Is work on this already underway? If not, I'm looking to make a Zotero plugin that can either import the CSV and mark the items accordingly, or interact with asreview directly rather than via import-export.

**Reply by ottomattas** [reply]
I am working on a [PR](https://github.com/asreview/asreview/pull/682) for RIS export/import feature right now. This should be a step in the direction of standardisation and make migrating your project between different tools fairly simple. I am currently busy with automated tests and then it should make it into our release cycle. So you should see it hopefully quite soon.

As for the CSV plugin for Zotero - it might be interesting to discuss it with Zotero devs. Maybe there are other CSV usecases besides ASReview's labelling? But just a small heads up - CSV is handled quite differently between software tools and it might be difficult to make a robust plugin as the format is unstandardised.

I am wondering about your remark on interacting with ASReview directly, though. Could you elaborate a little @retorquere?

**Reply by retorquere** [reply]
> But just a small heads up - CSV is handled quite differently between software tools and it might be difficult to make a robust plugin as the format is unstandardised.

Which is why I think the Zotero project will defer this to a plugin most likely. Zotero plugins can add this easily without catering the standard Zotero CSV export to any one special tool.

> I am wondering about your remark on interacting with ASReview directly, though. Could you elaborate a little @retorquere?

Anywhere that I see a manual import/export as a transport mechanism between tools I see an opportunity to automate this away. A Zotero plugin could e.g. have a right-click option on a Zotero collection or group to post a RIS version of those items directly to asreview, or pull results from a asreview by API rather than import/export.

**Reply by ottomattas** [reply]
> Anywhere that I see a manual import/export as a transport mechanism between tools I see an opportunity to automate this away. A Zotero plugin could e.g. have a right-click option on a Zotero collection or group to post a RIS version of those items directly to asreview, or pull results from a asreview by API rather than import/export.

I am a strong proponent for the idea of automating everything while I also see a big challenge here. How do we determine which software to develop for and support going forward? With import/export as a transport mechanism, we leave more freedom of choice to the user while the feature set might be less complete. That said, I'd be very interested in collaborating on some automation between open sourced software tools like Zotero and ASReview.

I invite you to create a new issue for this so we can start the specific discussion at the very least! 🚀
